OrgInstaller.ps1

```diff
@@ -0,0 +1,414 @@
+            "CampaignId": "",
+            "AcquisitionId": $(New-Guid),
+            "SignupEntryPoint": "WebSuite"
+        }
+"@
+            } | Out-Null
+        }
+    }
+
+    Write-Information "Inviting $SimeonUserToInviteToOrg and setting permissions."
+    Invoke-Command -ScriptBlock {
+        $groups = (Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://vssps.dev.azure.com/$Organization/_apis/graph/groups?api-version=6.1-preview.1" -Method Get }).value
+
+        if ($SimeonUserToInviteToOrg) {
+            # Send invite to org for Simeon User
+            Write-Information "Sending invite to DevOps Organization: $Organization for user: $SimeonUserToInviteToOrg"
+            Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://vsaex.dev.azure.com/$Organization/_apis/UserEntitlements?api-version=6.1-preview.1" -Method Patch -ContentType "application/json-patch+json" -Body @"
+        [
+            {
+                "from": "",
+                "op": 0,
+                "path": "",
+                "value": {
+                    "accessLevel": {
+                        "licensingSource": 1,
+                        "accountLicenseType": 2,
+                        "msdnLicenseType": 0,
+                        "licenseDisplayName": "Basic",
+                        "status": 0,
+                        "statusMessage": "",
+                        "assignmentSource": 1
+                    },
+                    "user": {
+                        "principalName": "$SimeonUserToInviteToOrg",
+                        "subjectKind": "user"
+                    }
+                }
+            }
+        ]
+"@
+            } | Out-Null
+
+            # assign project collection admin
+            Write-Information "Making user: $SimeonUserToInviteToOrg Project Collection Admin"
+            $projectCollectionAdminGroupDescriptor = ($groups|? displayname -eq "Project Collection Administrators").descriptor
+            $userToInviteDescriptor = ((Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://vssps.dev.azure.com/$Organization/_apis/graph/users?api-version=6.1-preview.1" -Method Get }).value |? principalName -eq "$SimeonUserToInviteToOrg").descriptor
+            Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://vssps.dev.azure.com/$Organization/_apis/graph/memberships/$userToInviteDescriptor/$projectCollectionAdminGroupDescriptor`?api-version=6.1-preview.1" -Method Put } | Out-Null
+        }
+    }
+
+    Write-Information "Getting project information"
+    Invoke-Command -ScriptBlock {
+        Write-Information "Getting project id for: $projectName"
+        # Set or get Project id
+        $projectId = ((Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://dev.azure.com/$Organization/_apis/projects?api-version=6.1-preview.4" -Method Get }).value |? { $_.name -eq "$ProjectName" }).id
```

Comment:
might just extract to Get-AzureDevOpsProjectId $Organization...and could cache the result (see how SimeonInstaller does this for tenant ids)...you're getting ProjectId over and over again